### DISCUSSION TITLE
the creation of card_templates (an offshoot of button_card_templates but for styling all cards, which would obviously be a new project)

### DISCUSSION THREAD

**Discussion by jdtechsol** [custom-cards/button-card#954] [Ideas]
**Is your feature request related to a problem? Please describe.**
yes but only that the feature, or rather, addon, does not exist?

**Describe the solution you'd like**
button_card_templates to be made into its own HACS addon but for any card

**Describe alternatives you've considered**
card-mod but button_card_templates is better imho

**Additional context**
none

**Comment by Mariusthvdb** [comment]
Did you check decluttering-card?

**Comment by Mariusthvdb** [comment]
in all honesty: I feel this should be discontinued/closed as the obvious option would be card_mod for ll cards but button-card, optionally in  combination with decluttering-card, as mentioned above.
